includes/functions.php

```diff
@@ -2141,7 +2165,8 @@ function tsml_import_changes($feed_meetings, $data_source_url, $data_source_last
 		$feed_slugs[] = $meeting_slug;
 
 		// has the meeting been updated since the last refresh?
-		$current_meeting_last_update = strtotime($meeting['updated']);
+		$current_meeting_localised_last_update = tsml_date_localised(get_option('date_format') . ' ' . get_option('time_format'), strtotime($meeting['updated']));
+		$current_meeting_last_update = strtotime($current_meeting_localised_last_update);
```

Comment:
if we are using this to compare to a UNIX date below i don't think it should be localized. i think we can just do `time()` which should return the current unix timestamp

Reply:
Nope, not comparing to the current timestamp. We are comparing the last update field (which is UTC based) of the current record to the database timestamp of the last refresh which is stored as a localized datetime field.

Reply:
ah right, that makes sense. i see we set the last updated timestamp with `current_time('timestamp')` can we use the same method to compare it - that way we are sure we are comparing the same thing